src/main/scala/Core/util.scala

```diff
@@ -9,6 +9,7 @@ package flexpret.util
 
 import chisel3._
 import chisel3.util.BitPat
+import scala.language.implicitConversions
```

Comment:
Is this some kind of marker interface? It doesn't look like anything is used from this package, unless there's some kind of operator overloading going on...

Reply:
Yes, this is a marker interface. An implicit function is defined in `util.scala`. Scala gives a warning if this is not set.

implicit conversions are powerful, but might also reduce readability. I am not a big fan of it.